src/Http/Controllers/MailSendController.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Xammie\Mailbook\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\Mail\Mailable;
+use Illuminate\Mail\Mailer;
+use Xammie\Mailbook\Facades\Mailbook;
+use Xammie\Mailbook\MailableItem;
+
+class MailSendController
+{
+    public function __invoke(Mailer $mailer, Request $request): string
+    {
+        $request->validate([
+            'email' => 'email|required',
+            'item' => 'required',
+        ]);
+        /** @var string $item */
+        $item = $request->get('item');
+
+        if (! is_subclass_of($item, Mailable::class)) {
```

Comment:
This will not work for notifications which are also supported. You can look at how the mailable is resolved in the `MailContentController`

Reply:
Fixed it!